sunpy/timeseries/sources/noaa.py

```diff
@@ -154,6 +177,42 @@ def _parse_file(cls, filepath):
             # TODO: fix header/meta, it's returning rubbish.
             return data, MetaDict({'comments': header}), units
 
+    @staticmethod
+    def _parse_json_file(filepath):
+        """
+        Parses an NOAA indices JSON file.
+
+        Parameters
+        ----------
+        filepath : `str`
+            The path to the file you want to parse.
+        """
+        with open(filepath) as fp:
+            fp.seek(0)
+            data = pd.read_json(fp.read())
+
+        rename = {'ssn': 'mean monthly S.I.D.C. sunspot number',
+                  'smoothed_ssn': 'smoothed S.I.D.C. sunspot number',
+                  'observed_swpc_ssn': 'mean monthly SWPC/SWO sunspot number',
+                  'smoothed_swpc_ssn': 'smoothed SWPC/SWO sunspot number',
+                  'f10.7': 'mean monthly 10.7cm radio flux',
+                  'smoothed_f10.7': 'smoothed 10.7cm radio flux'}
```

Comment:
These column names are long. I don't really have a strong opinion but it leads to a lot of typing when accessing them?

Reply:
I'd be pro not renaming, and making sure there's a link to the external dataset description (that hopefully explains what these are) in the sunpy docs.

Reply:
But are they the same as before?

Reply:
Sorry I didn't realise, that's definitely a good reason to keep the long names (for backcompat)!

Reply:
I'm asking, I don't know. I am happy to keep them the same or map them but I don't want to do it blindly.

Reply:
I think we should keep them the same, so users don't have to change their code to update the names they are indexing the data with.

Reply:
I have renamed both sets back to normal.